### ISSUE TITLE
Rendering freeze after alt-tabbing from fullscreen on Linux with NVIDIA

### ISSUE THREAD

**Issue by Gert-dev** [mvdevs/jk2mv#132]
First of all, thanks for this project!

When running in fullscreen mode, if I alt-tab out of the game, and then back in, rendering "freezes". The game _does_ appear to keep running properly - you still hear sound and music from menu's and input, it just visually isn't updated anymore.

This doesn't happen if I run in windowed mode, only with fullscreen mode.

I'm running Arch Linux 64-bit with GNOME shell 3.34 and NVIDIA driver 440.59.

**Comment by aufau** [comment]
What version of jk2mv are you using exactly? You can check it for example in the bottom right of the in-game console. Or by checking JK2MV cvar value.

**Comment by aufau** [comment]
Regardless, please try the latest build. Similar bug was reported some time ago and it was fixed in the master branch. Unfortunately the fix isn't present in 1.4.1 and we were unable to release another version since then, even though there are substantial changes in the repository.

**Comment by Gert-dev** [comment]
The console says jk2mv vUNKNOWN, but I installed from the AUR package from Arch and updated it to use the 1.4.1 download from the releases page (since it's stuck at 1.2.1).

I tried [a build from CI](https://dl.bintray.com/mvdevs/jk2mv/jk2mv-v1.4-183-gb44a99b-linux-amd64-portable.tar.gz) for version b44a99b, which appears to be the latest commit on `master` at the time of writing.

This indeed seems to fix the issue. Good to know that it's already been fixed. I'll close this issue and wait for the next stable release, in that case.

Thanks for your efforts!
